Dockerfile

```diff
@@ -46,7 +46,6 @@ COPY --from=golang_121 /usr/local/go /usr/local/go/go1.21
 COPY --from=node_223 /usr/local/lib/node_modules/corepack /usr/local/lib/corepack
 COPY --from=node_223 /usr/local/bin/node /usr/local/bin/node
 COPY --from=builder /venv /venv
-COPY --from=builder /src/utils/merge_syft_sbom.py /usr/local/bin/merge_syft_sbom
```

Comment:
**Commit message:**

Nitpicking about the commit message, let's improve the wording just a bit. How about:
_Removing the script from this repo and migrating it to a place and organization more appropriate to host this scrip than this project, see https://github.com/konflux-ci/build-tasks-dockerfiles/pull/154_

Reply:
Erik, please :D